### ISSUE TITLE
Adding external table to sql 2019 fails

### ISSUE THREAD

**Issue by dawazualex** [Microsoft/azuredatastudio#2819]
Issue Type: <b>Bug</b>

Try to add an Oracle external table, after attempting to create a connection to my data source, I get an error "external data sources are not supported with type GENERIC"

Azure Data Studio version: azuredatastudio 1.0.0 (cab8f3e37c9ac4367e0dacef5185da08bd8021c2, 2018-09-20T22:04:35.897Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz (4 x 3600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.90GB (20.58GB free)|
|Process Argv|D:\Program Files\Azure Data Studio\azuredatastudio.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
mssql-db-insights|mat|0.2.2
agent|Mic|0.33.0
import|Mic|0.2.0
server-report|Mic|0.1.3
sql-vnext|Mic|0.6.9

</details>

**Comment by corivera** [comment]
Have you enabled PolyBase in your SQL Server instance?
https://docs.microsoft.com/en-us/sql/relational-databases/polybase/polybase-installation?view=sql-server-2017

**Comment by dawazualex** [comment]
Yes I did.  Still get that error. And yes I verified I installed the CTP
2.0.  Installed all the features

**Comment by corivera** [comment]
Are you trying to create this external table through T-SQL, or the External Table wizard in the sql-vnext extension?

**Comment by dawazualex** [comment]
Both. I get the same error on both.  Happy the share my desktop some time?

**Comment by corivera** [comment]
@dawazualex Were you able to resolve this issue?

**Comment by dawazualex** [comment]
I figured it out.  It was an edition issue.  Trying to connect to an oracle
data source with a ton of tables (1400 +), I'm getting this error when
expanding the database to see the list of tables - The user specified
schema column count is 55, which varies from the external table schema
column count of 65.

**Comment by corivera** [comment]
Could you open a separate issue for that bug?

**Comment by AdeDarbs** [comment]
I have a similar issue when trying to Create an External Data Source using TSQL  in SQL Server 2019 for the purpose of connecting to MongoDB through PolyBase.

"External data sources are not supported with type GENERIC."

ODBC Drivers installed. 

I am thinking that due to SQL Server 2019 being in CTP it may not be working yet.

**Comment by kevcunnane** [comment]
@AdeDarbs that error message indicates Polybase not being enabled. Per the [instructions](https://docs.microsoft.com/en-us/sql/relational-databases/polybase/polybase-installation?view=sql-server-2017), install & enable Polybase. In particular you'll want to run the enable polybase command (see below).
 
```
exec sp_configure @configname = 'polybase enabled', @configvalue = 1;
RECONFIGURE [ WITH OVERRIDE ]  ;
```

**Comment by AdeDarbs** [comment]
Hey Kev. My script does that. Polybase is installed. It must be a SQL config setting surrounding TCP/IP and the 27017 port. Thanks for replying.
